packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import ThemeContext from 'terra-theme-context';
+import classNames from 'classnames/bind';
+import '../_elementPolyfill';
+import styles from './Row.module.scss';
+import RowSelectionCell from './RowSelectionCell';
+import Cell from './Cell';
+import rowShape from '../proptypes/rowShape';
+import WorklistDataGridPropTypes from '../proptypes/WorklistDataGridPropTypes';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The row's position in the Grid. This is zero based.
+   */
+  rowIndex: PropTypes.number,
+  /**
+   * String that specifies height of the row. Any valid CSS width value is accepted.
+  */
+  height: PropTypes.string,
+  /**
+   * The information about the Row.
+   */
+  row: rowShape.isRequired,
+  /**
+   * Boolean indicating whether or not the DataGrid should allow entire rows to be selectable. An additional column will be
+   * rendered to allow for row selection to occur.
+   */
+  hasSelectableRows: PropTypes.bool,
```

Comment:
The prop name `hasSelectableRows` is misleading; It seems like the row itself is selectable, not its children. We can therefore rename this to `isSelectable`.

Reply:
Personally, I prefer the 'isRowSelectionMode' but we kept the same name as the original clinical data grid and used it throughout for consistency. Let me know if you still think we should change it.

Reply:
As agreed, we changed this to hasRowSelection